src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {
+    void draw(Figures figure);
```

Comment:
```suggestion
    void draw();
```
метод може нічого не приймати на вхід, фігура сама себе хай малює